src/harness/util.py

```diff
@@ -376,3 +375,22 @@ def addGrantIdsToRequests(grant_ids, requests):
     requests: (list) list of requests, containing dictionaries.
   """
   addIdsToRequests(grant_ids, requests, 'grantId')
+
+def getCertFilename(cert_name):
+  """Returns the absolute path of the file corresponding to the given |cert_name|.
+  """
+  harness_dir = os.path.dirname(os.path.abspath
+                                (inspect.getfile(inspect.currentframe())))
+  return os.path.join(harness_dir, 'certs', cert_name)
+
+def getCertificateFingerprint(certificate):
+  """ Get SHA1 hash of the input certificate.
+  Args:
+    certificate: certificate file
```

Comment:
Previous review: "It's not really a file, is it?"

> Updated the comments correctly.

This does not appear to be done.

Reply:
Updated   the doc string to note that it is the full path to the file.